code/frameworks/svelte-vite/package.json

```diff
@@ -54,13 +54,13 @@
     "svelte2tsx": "^0.7.13",
     "sveltedoc-parser": "^4.2.1",
     "ts-dedent": "^2.2.0",
-    "typescript": "^4.9.4 || ^5.0.0"
+    "typescript": "^5.7.3"
```

Comment:
`typescript` is defined here as a `dependency`. This may affect users.

Reply:
Thank you, that's a great catch! I was looking out for such, but I missed this.